packages/scripts/test/optimistic-oracle-umip/1_Propose.ts

```diff
@@ -0,0 +1,102 @@
+// This script generates and submits an upgrade transaction to add/upgrade the optimistic oracle in the DVM. It can be
+// run on a local hardhat node fork of the mainnet or can be run directly on the mainnet to execute the upgrade
+// transactions. To run this on the localhost first fork mainnet into a local hardhat node by running:
+// HARDHAT_CHAIN_ID=1 yarn hardhat node --fork https://mainnet.infura.io/v3/<YOUR-INFURA-KEY> --port 9545 --no-deploy
+// Then execute the script from core:
+// OPTIMISTC_ORACLE_V2=<OPTIMISTC_ORACLE_V2_ADDRESS> yarn workspace @uma/scripts hardhat run ./test/optimistic-oracle-umip/1_Propose.ts --network localhost
```

Comment:
does running `yarn hardhat run ...` directly from the `scripts` package work without having to define the `yarn workspace` part?

Reply:
yes, that also work. I did it like this to be consistent with how the rest of scripts are documented:
`NODE_URL_1=http://127.0.0.1:9545/ node ./packages/scripts/src/admin-proposals/simulateVote.js --network mainnet-fork`

But I agree it makes sense to run them from the root of the package, specially if some day we plan to split the monorepo. I can update all the scripts commands of the readmes